features/location/api/build.gradle.kts

```diff
@@ -37,9 +37,23 @@ android {
             type = "string",
             name = "maptiler_api_key",
             value = System.getenv("ELEMENT_ANDROID_MAPTILER_API_KEY")
-                ?: readLocalProperty("services.maptiler.apikey") as? String
+                ?: readLocalProperty("services.maptiler.apikey")
                 ?: ""
         )
+        resValue(
+            type = "string",
+            name = "maptiler_light_map_id",
+            value = System.getenv("ELEMENT_ANDROID_MAPTILER_LIGHT_MAP_ID")
+                ?: readLocalProperty("services.maptiler.lightMapId")
+                ?: "basic-v2" // fall back to maptiler's default map.
+        )
+        resValue(
+            type = "string",
+            name = "maptiler_dark_map_id",
+            value = System.getenv("ELEMENT_ANDROID_MAPTILER_DARK_MAP_ID")
+                ?: readLocalProperty("services.maptiler.darkMapId")
+                ?: "basic-v2" // fall back to maptiler's default map.
```

Comment:
basic-v2-dark?

Reply:
OH indeed it works but it was not shown in maptiler's dashboard. How did you figure it out?

Reply:
It's a bit buried: https://www.maptiler.com/maps/dark/ has a "Use Basic Dark" link and the style ID is in the URL. (I think I'd seen it before, otherwise I wouldn't have known to gone hunting there!)